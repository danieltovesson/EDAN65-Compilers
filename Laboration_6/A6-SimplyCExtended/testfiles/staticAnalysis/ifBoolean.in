int main() {
  if (1 < 2) {
    int a = 1;
  }
  if (2 > 1) {
    int b = 2;
  }
  if (1 <= 1) {
    int a = 1;
  }
  if (2 >= 1) {
    int b = 2;
  }
  if (1 != 2) {
    int a = 1;
  }
  if (1 == 1) {
    int b = 2;
  }
  if (2 < 1) {
    int a = 1;
  }
  if (1 > 2) {
    int b = 2;
  }
  if (2 <= 1) {
    int a = 1;
  }
  if (1 >= 2) {
    int b = 2;
  }
  if (2 != 2) {
    int a = 1;
  }
  if (2 == 1) {
    int b = 2;
  }
}
