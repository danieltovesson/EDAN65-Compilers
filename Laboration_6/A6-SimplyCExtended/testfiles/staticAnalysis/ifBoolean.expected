If condition: 1
Assignment: main0_0_a = 1
If condition: 1
Assignment: main1_0_b = 2
If condition: 1
Assignment: main2_0_a = 1
If condition: 1
Assignment: main3_0_b = 2
If condition: 1
Assignment: main4_0_a = 1
If condition: 1
Assignment: main5_0_b = 2
If condition: 0
If condition: 0
If condition: 0
If condition: 0
If condition: 0
If condition: 0
